src/common/util.tsx

```diff
@@ -80,3 +82,21 @@ export const getLocalFocusData = (): string[] => {
   }
   return JSON.parse(localData) as string[];
 };
+
+export const getLocalQuestListDepthData = (): number => {
+  const localQuestListDepth = localStorage.getItem("questDepth");
+  if (localQuestListDepth) {
+    return +localQuestListDepth;
+  }
+
+  return 0;
```

Comment:
May as well totally code golf it
```suggestion
  return localQuestListDepth ?? +localQuestListDepth : 0;
```